src/js/helpers.js

```diff
@@ -578,3 +580,17 @@ export function omitRequired(schema) {
 
   return newSchema;
 }
+
+/*
+ * Normal transform for schemaform data
+ */
+export function transformForSubmit(formConfig, form, replacer = stringifyFormReplacer) {
+  const expandedPages = expandArrayPages(createFormPageList(formConfig), form.data);
+  const activePages = getActivePages(expandedPages, form.data);
```

Comment:
Why are we not using `getActiveExpandedPages` here if it's doing these same 2 checks in this order? Because we need to calculate `expandedPages` separately for the next function?

Reply:
Yeah, I thought it was useful to used the expandedPage array for both of the following function calls and make it clear that we're using the same base data to get the active and inactive page lists.

Reply:
Yep, that's what it seemed like. Feel free to merge this when you're ready @jbalboni!